src/commands/project/create.ts

```diff
@@ -4,26 +4,54 @@ import { Arg } from "@oclif/core/lib/interfaces";
 import { TransactionCommand } from "../../base";
 import { getContract, getSigner, parseAddress, parseHash, pretty, run } from "../../helpers";
 
+interface ProjectMetadata {
+  type: "static" | "nextjs";
+  port?: number;
```

Comment:
since port/start command are consistent and internal should these be configurable?